Answers.md

```diff
@@ -1,2 +1,35 @@
 # Your responses to the short answer questions should be laid out here using Mark Down.
 ### For help with markdown syntax [Go here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+
+# 1. Describe some of the differences between `.forEach` & `.map`. 
+     
+     .forEach is a loop that applies an operation to each list item. ForEach does not return something.
+     .map is a loop that creates a new list after performing a transformative task on each list item. Map returns something.
+
+# 2. Name five different Types in JavaScript. A Type is something that can represent data. What is so special about Arrays?
+     
+     String
+     Number
+     Boolean
+     Null
+     Undefined
+     Symbols (added in ES6)
+
+     Arrays can store multiple data types.
```

Comment:
I'm looking here for this: to add on top of what you've said. Arrays are special types of Objects in JS.